config/state_machine/configs/00_moj/60_sar_internal_review/30_sar_internal_review_trigger_workflow.yml

```diff
@@ -0,0 +1,249 @@
+
+              awaiting_dispatch:
+                add_message_to_case:
+                  if: Workflows::Predicates#responder_is_member_of_assigned_team?
+                  after_transition: Workflows::Hooks#notify_responder_message_received
+                respond_and_close:
+                  if: Workflows::Predicates#responder_is_member_of_assigned_team?
+                  transition_to: closed
+                respond:
+                  if: Workflows::Predicates#responder_is_member_of_assigned_team?
+                close:
+                  after_transition: Workflows::Hooks#notify_managing_team_case_closed
+                  if: Workflows::Predicates#responder_is_member_of_assigned_team?
+                  transition_to: closed
+                reassign_user:
+                  if: Workflows::Predicates#responder_is_member_of_assigned_team?
+                  after_transition: Workflows::Hooks#reassign_user_email
+
+              closed:
+                add_message_to_case:
+                  if: Workflows::Predicates#responder_is_member_of_assigned_team?
+                update_closure:
+                  if: Workflows::Predicates#overturned_editing_enabled_and_responder_in_assigned_team?
+
+          ########################### SAR INTERNAL REVIEW :: STANDARD WORKFLOW :: APPROVER ########################
+          approver:
+            states:
+              unassigned:
+                accept_approver_assignment:
+                  if: Case::BasePolicy#can_accept_or_reject_approver_assignment?
+                add_message_to_case:
+                  if: Workflows::Predicates#user_is_approver_on_case?
+                reassign_user:
+                  if: Workflows::Predicates#case_is_assigned_to_responder_or_approver_in_same_team_as_current_user
+                  after_transition: Workflows::Hooks#reassign_user_email
+                unaccept_approver_assignment:
+                  if: Case::BasePolicy#can_unaccept_approval_assignment?
+                unflag_for_clearance:
+                  if: Workflows::Predicates#case_can_be_unflagged_for_clearance?
+                  switch_workflow: standard
+
+              awaiting_responder:
+                accept_approver_assignment:
+                  if: Case::BasePolicy#can_accept_or_reject_approver_assignment?
+                add_message_to_case:
+                  if: Workflows::Predicates#user_is_approver_on_case?
+                reassign_user:
+                  if: Workflows::Predicates#case_is_assigned_to_responder_or_approver_in_same_team_as_current_user
+                  after_transition: Workflows::Hooks#reassign_user_email
+                unaccept_approver_assignment:
+                  if: Case::BasePolicy#can_unaccept_approval_assignment?
+                unflag_for_clearance:
+                  if: Workflows::Predicates#case_can_be_unflagged_for_clearance?
+                  switch_workflow: standard
```

Comment:
I can see there is a button to switch the trigger-flow back to standard,  based on my understanding, we shouldn't have such switching function, but anyway you can check with Nicole 


Actually,  there is something else coming to my mind, that there is another option is that no need define a duplicated workflow of SAR existing flow, just point to the same flow,   by using `def self.state_machine_name`

Reply:
🤔 that's a good idea I'll look into that.

Reply:
I have stuck with the original workflow and amended the predicate 👍